third_party/move/tools/move-formatter/tests/complex/branch/input2.fmt.move

```diff
@@ -0,0 +1,8 @@
+module test {
+    fun f(x: u64): u64 {
+        if (x == 1) {if (x == 11) { 11 }}
```

Comment:
I am not sure whether it is good idea to reformat the `if` block into one line here. cc @vineethk

Reply:
Does rustfmt do the same thing?

Reply:
rustfmt will increase into new lines, regardless of whether the block contains a simple statement or not. We will adjust the processing logic here to be consistent with rustfmt

Reply:
Looks like this is being resolved to what `rustfmt` does, which is likely the better choice.